typescript/sdk/src/metadata/chainMetadataTypes.ts

```diff
@@ -30,7 +31,7 @@ export enum ChainTechnicalStack {
   OpStack = 'opstack',
   PolygonCDK = 'polygoncdk',
   PolkadotSubstrate = 'polkadotsubstrate',
-  ZkSync = 'zksync',
+  ZKSync = 'zksync',
```

Comment:
nit: can we we keep this as `ZkSync` and update all the `ZKSync` -> `ZkSync` for naming elsewhere as well for consistency?

imo it should generally be `ZkXYZ` rather than `ZKXYZ`, @ltyu wdyt?

Reply:
We sure can, but zksync did the rebranding and is actually the owner of "ZK" trademark, so at the end the value should be ZKsync. But at the end it really doesn't matter, whatever you guys prefer